<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1250</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1250</width>
    <height>800</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sniffer</string>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1231</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="TophorizontalLayout">
       <item>
        <widget class="QLabel" name="chooseNIClabel">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>     选择网卡</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="chooseNICComboBox">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="currentText">
          <string>TEST</string>
         </property>
         <item>
          <property name="text">
           <string>TEST</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startCaptureBtn">
         <property name="text">
          <string>开始抓包</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopCapture">
         <property name="text">
          <string>停止抓包</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearDataBtn">
         <property name="text">
          <string>清空数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveDataBtn">
         <property name="text">
          <string>保存数据</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readDataBtn">
         <property name="text">
          <string>读取数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="MidhorizontalLayout">
       <item>
        <widget class="QLabel" name="proTypeLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>协议类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="proTypeChooseBox">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>ALL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ARP ONLY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TCP ONLY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UDP ONLY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>TCP OR UDP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>IP ONLY</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="srcIPLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>源IP地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="srcIPLineEdit">
         <property name="text">
          <string>112.123.414.212</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="srcPortLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>源端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="srcPortLineEdit">
         <property name="text">
          <string>1234</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="desIPLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>目的IP地址</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="desIPLineEdit">
         <property name="text">
          <string>112.123.414.212</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="desPortLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>目的端口</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="desPortLineEdit">
         <property name="text">
          <string>3123</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filterBtn">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设置过滤</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="analysisHLayout">
       <item>
        <widget class="QLabel" name="analysisLabel">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>流量统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ProtoCntBnt">
         <property name="text">
          <string>协议统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inFlowCntBnt">
         <property name="text">
          <string>流入流量统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="outFlowCntBnt">
         <property name="text">
          <string>流出流量统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="flowTimeCntBnt">
         <property name="text">
          <string>流量时间统计</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ipMapBnt">
         <property name="text">
          <string>IP归属地</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="pktInfoTable">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="columnCount">
        <number>7</number>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>No.</string>
        </property>
        <property name="font">
         <font>
          <strikeout>false</strikeout>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Src.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Des.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Protocol Type</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Length(bytes)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Info</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="BomhorizontalLayout">
       <item>
        <widget class="QTextEdit" name="pktDetailWin">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-right:5px solid #ECECEC;border-top:2px solid #ECECEC</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑','微软雅黑'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'微软雅黑';&quot;&gt;TEST&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="hexDumpWin">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-top:2px solid #ECECEC</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;TEST&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
